src/js/controller/controller.js

```diff
@@ -585,7 +586,11 @@ Object.assign(Controller.prototype, {
 
         function _item(index, meta) {
             _stop(true);
-            _setItem(index);
+            _setItem(index)
+                .catch(e => {
```

Comment:
this looks good but my concern is you can switch items by calling `load(1)` which would switch to item index 2 which would directly call `_setItem`. I feel like loading an index should call `_item`

Reply:
`load()` should not `stop` and `play` before and after changing the item. It doesn't with `load(feed)` or `load(object)` so the same applies to `load(Number)`.

Reply:
@robwalch why do we have a different implementation for `load(Number)` and `next()`? Shouldn't `load(currentIndex+1)` be the same as calling `next()`?